shardingsphere-sql-parser/shardingsphere-sql-parser-test/src/main/resources/case/dml/delete.xml

```diff
@@ -366,7 +366,7 @@
                     <column name="order_id" start-index="10" stop-index="17"/>
                     <column name="user_id" start-index="20" stop-index="26"/>
                 </columns>
-                <subquery-expression>
+                <subquery-expression start-index="32" stop-index="70">
```

Comment:
Why do you modify delete.xml ?

Reply:
Since I tried to assert the `SubquerySegment` of `CommonTableExpressionSegment` in `WithClauseAssert`, I needed to provide start and stop indices for all `subquery-expression`s within `common-table-expression`s as it was asserted.